types/localizejs/index.d.ts

```diff
@@ -0,0 +1,225 @@
+// Type definitions for LocalizeJS 1.0
```

Comment:
If I'm right, the name should be same with the package name.

Reply:
There is no package available for this library on NPM. It is added using a snippet and exposes a global variable. Would you suggest I use a different name?